aiofreepybox/api/home.py

```diff
@@ -0,0 +1,266 @@
+# Home structure : adapter > node > endpoint
+class Home:
+
+    def __init__(self, access):
+        self._access = access
+
+    home_endpoint_value_schema = {
+        'value': object
+    }
+
+    create_home_node_rule_payload_schema = {
+        'iconUrl': '',
+        'id': int,
+        'label': '',
+        'name': '',
+        'role': int,
+        'roleLabel': '',
+        'type': ''
+    }
+
+    node_rule_role_schema = {
+        'node': [int],
+        'role': int
+    }
+
+    node_rule_configuration_data_schema = {
+        'roles': [node_rule_role_schema]
+    }
+
+    sms_number_data_schema = {
+        'description': 'Mon numero',
+        'phoneNumber': '',
+        'smsEnabled': bool(True),
+        'voicemailEnabled': bool(True)
+    }
+
+    sms_validation_data_schema = {
+        'applicationHash': ''
+    }
+
+    sms_number_validation_data_schema = {
+        'validationCode': ''
+    }
+
+    next_pairing_step_payload_schema = {
+        'session': int,
+        'pageid': int,
+        'fields': [object]
+    }
+
+    start_pairing_step_payload_schema = {
+        'nfc': bool(True),
+        'qrcode': bool(),
+        'type': ''
+    }
+
+    stop_pairing_step_payload_schema = {
+        'session': int,
+    }
+
+    async def del_home_adapter(self, home_adapter_id):
+        '''
+        Delete home adapter
+        '''
+        return await self._access.delete(f'home/adapters/{home_adapter_id}')
+
+    async def get_home_adapter(self, home_adapter_id):
+        '''
+        Retrieve a registered home adapter
+        '''
+        return await self._access.get(f'home/adapters/{home_adapter_id}')
+
+    async def get_home_adapters(self):
+        '''
+        Retrieve the list of registered home adapters
+        '''
+        return await self._access.get('home/adapters')
+
+    async def get_camera(self):
+        '''
+        Get camera info
+        '''
+        return await self._access.get('camera')
+
+    async def get_camera_snapshot(self, camera_index=0, size=4, quality=5):
```

Comment:
Could you please explain how the size and quality values should be specified? Just a comment could be sufficient, or maybe a bunch of named constants if only some values are valid?

Reply:
Added comment:
        `size`: 2 = 320x240, 3 = 640x480, 4 = 1280x720
        `quality`: quality index, default is 5